requirements/torch_cu102_linux.txt

```diff
@@ -1,5 +1,5 @@
 # pip install torch torchvision
-torch==1.9.0; sys_platform == 'linux'
-torchvision==0.10.0; sys_platform == 'linux'
-torchtext==0.10.0; sys_platform == 'linux'
-torchaudio==0.9.0; sys_platform == 'linux'
\ No newline at end of file
+torch==1.10.0+cu102; sys_platform == 'linux'
+torchvision==0.11.1+cu102; sys_platform == 'linux'
+torchtext==0.11.0; sys_platform == 'linux'
+torchaudio==0.10.0+cu102; sys_platform == 'linux'
```

Comment:
gpu build failed b/c torchvision==0.11.1+cu102 does not exist.

it should be:
torch==1.10.0; sys_platform == 'linux'
torchvision==0.11.1; sys_platform == 'linux'
torchtext==0.11.0; sys_platform == 'linux'
torchaudio==0.10.0; sys_platform == 'linux'